app/src/main/java/com/bilginyuksel/sdk/push/MainActivity.java

```diff
@@ -3,38 +3,24 @@
 import androidx.annotation.RequiresApi;
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Build;
 import android.os.Bundle;
 import android.widget.Button;
 
-import com.bilginyuksel.sdk.push.api.ClientServerConnection;
-import com.neovisionaries.ws.client.WebSocketException;
+import com.bilginyuksel.sdk.push.api.MessageService;
+import com.bilginyuksel.sdk.push.api.WebSocketListenerImpl;
+import com.bilginyuksel.sdk.push.api.WsConnection;
 
-import java.io.IOException;
-import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
 
 public class MainActivity extends AppCompatActivity {
 
-    public class RunnerThread extends Thread {
-
-        @Override
-        public void run() {
-            ClientServerConnection connection = new ClientServerConnection(getApplicationContext());
-            try {
-                connection.connect();
-            } catch (IOException | URISyntaxException | WebSocketException e) {
-                e.printStackTrace();
-            }
-        }
-    }
-
     @RequiresApi(api = Build.VERSION_CODES.O)
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        NotificationSender notificationSender = new NotificationSender(getApplicationContext());
-        setContentView(R.layout.activity_main);
-        Button button = findViewById(R.id.sendNotifbutton);
-        button.setOnClickListener(v -> notificationSender.send());
+        startService(new Intent(this, MessageService.class));
```

Comment:
Maybe we can run the service automatically via `AndroidManifest.xml` configurations.